priv/templates/nimble_template/.github/workflows/README.md.eex

```diff
@@ -1,13 +1,14 @@
 # Github Actions
 
-The following workflows are supported. Depends on what you choose when generating the project, some or all of them would be available:
+The following workflows are supported.
 
+<%= if with_test_workflow? do %>
 - Test
-- [Deploy to Heroku](#deploy-to-heroku-workflow)
+<% end %>
+<%= if with_deploy_to_heroku_workflow? do %>
+- [Deploy to Heroku](#deploy-to-heroku-workflow-usage-instruction)
```

Comment:
I think this part should be wrapped in its own `if` clause because this section is for the Worklow list 
so we can have more workflows added later
```suggestion
<%= if with_deploy_to_heroku_workflow? do %>
- [Deploy to Heroku](#deploy-to-heroku-workflow-usage-instruction)
<% end %>

# and later we can have
<%= if with_deploy_to_ecs_workflow? do %>
- [Deploy to ECS](#deploy-to-ECS-workflow-usage-instruction)
<% end %>
```

Reply:
Yeah @longnd I was thinking about that, but for now, I just follow what we have in the current Readme content also I'm not sure we will have both Heroku and ECS content, but anw I will separate it to make it easy to edit in the future

Reply:
Adjusted in https://github.com/nimblehq/elixir-templates/pull/141/commits/0fd1ca2c51a7b792855bce98b0a42b003714e8fa